File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/tablets/RightTabletActivity.java
package com.jdroid.android.sample.ui.tablets;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class RightTabletActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return RightTabletFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/tablets/RightTabletActivity.kt
package com.jdroid.android.sample.ui.tablets

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class RightTabletActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return RightTabletFragment::class.java
    }
}
